schema/src/main/jade-tables/step_run.table.json

```diff
@@ -1,30 +1,25 @@
 {
-  "name": "pipeline_activity",
+  "name": "step_run",
   "columns": [
     {
       "name": "id",
       "datatype": "string",
       "type": "primary_key"
     },
     {
-      "name": "cross_references",
+      "name": "version",
       "datatype": "string",
-      "type": "repeated"
-    },
-    {
-      "name": "time_created",
-      "datatype": "timestamp",
-      "type": "required"
-    },
-    {
-      "name": "time_started",
-      "datatype": "timestamp",
       "type": "required"
     },
     {
-      "name": "time_ended",
-      "datatype": "timestamp",
-      "type": "required"
+      "name": "pipeline_run_id",
+      "datatype": "string",
+      "links": [
+        {
+          "table_name": "pipeline_run",
+          "column_name": "id"
+        }
+      ]
```

Comment:
Is this the pipeline_run_id or the step_run_id?  I think the latter.

Reply:
This is the `pipeline_run_id` (a foreign key referencing the pipeline run that this step run was a part of). I might be misunderstanding what you're asking though. Let me know if that doesn't answer the question.

Reply:
I didn't think we had the pipeline_run in Encode -- just the step_run for each step in the pipeline.